modules/runner/src/testFixtures/java/org/apache/ignite/internal/ClusterPerClassIntegrationTest.java

```diff
@@ -185,6 +185,42 @@ protected static void insertPersons(String tableName, Person... people) {
         );
     }
 
+    /**
+     * Updates data in the table created by {@link #createZoneAndTable(String, String, int, int)}.
+     *
+     * @param tableName Table name.
+     * @param people People to update in the table.
+     */
+    protected static void updatePersons(String tableName, Person... people) {
+        Transaction tx = CLUSTER.node(0).transactions().begin();
+
+        String sql = String.format("UPDATE %s SET NAME=?, SALARY=? WHERE ID=?", tableName);
+
+        for (Person person : people) {
+            sql(tx, sql, person.name, person.salary, person.id);
+        }
+
+        tx.commit();
+    }
+
+    /**
+     * Deletes data in the table created by {@link #createZoneAndTable(String, String, int, int)}.
+     *
+     * @param tableName Table name.
+     * @param personIds Person IDs to delete.
+     */
+    protected static void deletePersons(String tableName, int... personIds) {
```

Comment:
It's a bit strange that these methods concerning some Person class are defined in a generic integration test class, they don't seem to belong here. I suggest moving them to another class (maybe as static methods, or this new helper might be instantiated in each test class that actually needs it).

Reply:
I don’t see the need to do this yet, let them stay here for now, then if the situation is too much, we’ll do it better.

Reply:
But they are only used in one integration test, is this true? If yet, then why keep them here?

Reply:
For now, yes, but I have plans to continue using them, I won’t change them for now, if necessary, we’ll change them later.